common-libs/did-auth-lib/src/shared/builder.ts

```diff
@@ -0,0 +1,15 @@
+import { JwtService } from '@affinidi/tools-common'
+import { parse } from 'did-resolver'
+
+export const buildResponseJwtObject = async (didAuthRequestToken: string) => {
+  const didAuthRequestTokenDecoded = JwtService.fromJWT(didAuthRequestToken)
+  const jwtType = 'DidAuthResponse'
+  const NOW = Date.now()
```

Comment:
Why screaming? :smile:

Reply:
It is not a new code, just moved: https://github.com/affinityproject/affinidi-core-sdk/blob/a6de74683e066e238828cc679bb87c9c2beef631/common-libs/did-auth-lib/src/DidAuthService/DidAuthService.ts#L103